comparison/loadtest.go

```diff
@@ -232,21 +305,14 @@ func initLoadTest(t *terraform.Terraform, buildCfg BuildConfig, dumpFilename str
 		Value:   clearLicensesCmdValue,
 	}
 
-	resetBucketCmds := []localCmd{}
-	if s3BucketURI != "" && tfOutput.HasS3Bucket() {
-		deleteBucketCmd := localCmd{
-			msg:   "Emptying S3 bucket",
-			value: []string{"--profile", t.Config().AWSProfile, "s3", "rm", "s3://" + tfOutput.S3Bucket.Id, "--recursive"},
-		}
-
-		prepopulateBucketCmd := localCmd{
-			msg:   "Pre-populating S3 bucket",
-			value: []string{"--profile", t.Config().AWSProfile, "s3", "cp", s3BucketURI, "s3://" + tfOutput.S3Bucket.Id, "--recursive"},
-		}
-
-		resetBucketCmds = []localCmd{deleteBucketCmd, prepopulateBucketCmd}
+	// Should I use context background here?
+	cfg, err := config.LoadDefaultConfig(context.Background(), config.WithSharedConfigProfile(t.Config().AWSProfile))
+	if err != nil {
+		return fmt.Errorf("error loading aws config: %w", err)
 	}
 
+	bucket := s3ClientWrapper{S3Client: s3.NewFromConfig(cfg)}
```

Comment:
My previous comment was about the name of the variable itself, I think it's better to call this one `s3client` instead of `bucket`:
```suggestion
	s3client := s3ClientWrapper{S3Client: s3.NewFromConfig(cfg)}
```